torch/_inductor/sizevars.py

```diff
@@ -138,14 +115,16 @@ def visit_modular_indexing(base, divisor, modulus):
                 # actual iteration range is to size-1
                 iter_ranges_zero = {k: 0 for k, v in var_ranges.items()}
                 base_lowest = sympy_subs(base, iter_ranges_zero)
-                if self.maybe_guard_lt(base_lowest, 0):
+                if self.statically_known_leq(0, base_lowest):
                     # can't replace with indexing div if base can be negative
-                    return ModularIndexing(base, divisor, modulus)
+                    base_pos = True
+                else:
+                    base_pos = False
```

Comment:
What's going on here?

Reply:
@ngimel and I wrote this together, this is to invert it from lt to leq - we can't just return MI here, we ned to flag the base as positive to know if we can FD later.